### ISSUE TITLE
Add Cephalyx

### ISSUE THREAD

**Issue by Obiken** [BSData/warmahordes#24]

**Comment by JimOrtiz** [comment]
What do we think is the best way to approach adding Cephalyx?  A new catalog or adding the entries to Mercenaries & Cryx?  All of the above?  Adding the UA to all the existing units seems like it'd make "standard" Merc lists more cumbersome.

**Comment by Obiken** [comment]
I haven't seen the book yet, so not sure how they are supposed to be used.

But from what I've heard I think we need a new catalog for when you run them as a Cephalyx list. What I'm not sure on is if they can be added to regular Cryx or Mercenary lists. If so we either need to add the choices available to the existing catalogs, or we make duplicate catalogs with the entries in.

Might have to test it to see if adding them to the Cryx/Merc catalogs make them feel bloated and cumbersome.

**Comment by JimOrtiz** [comment]
Alright, new catalog up.  Both Contract & Theme option manipulate FA values, not really sure what to do about that.  All the FA values are based on the entries with no contract/theme-based changes.

Since in either version, non-Cephalyx units can't be taken without a Cephalyx UA, I set it as a mandatory addition on each unit.  

  I didn't close this, it feels like I can't use the catalog to build a list for either option, so it can't be right...  
Any thoughts?

**Comment by JimOrtiz** [comment]
Added two entries to the caster: "use contract" and "theme tier 1-4", and modifiers for FA allowances based on either contract or theme tier usage.  Theme tier 3 is "army contains units", but I couldn't figure out a modifier for that, so it's currently "selected items", so I bumped it by one to count the warcaster himself.  I opened a thread on the BattleScribe boards about it, but I felt like the catalog was complete enough to release.  Going to close this out.
